eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -87,7 +93,27 @@ def callback_full(theta, lc, model, prior1, prior2, priortype, freenames):
 
     # Get the best fit params
     fit_params = results.x
-
+    
+    # Create table of results
+    t_results = table.Table([freenames, fit_params], 
+                            names=("Parameter", "Mean")  ) 
+
+    # Save transmission spectrum
+    # indices of fitted rps for all channels
+    ind_spec = np.array([i for i in range(len(freenames)) if 'rp' in freenames[i]])
+    # get wavelengths at middle of bin
+    wave_low = meta.wave_low[lc.fitted_channels]
+    wave_hi = meta.wave_hi[lc.fitted_channels]
+    wave_mid = (wave_hi+wave_low)/2.
+    # Save transmission spectrum
+    # indices of fitted rps for all channels
+    ind_spec = np.array([i for i in range(len(freenames)) if 'rp' in freenames[i]])
+    if len(ind_spec):
```

Comment:
I had implemented it and presented it before for Stage 5 so I thought this was something that people would find useful in this stage as I for instance don't use Stage 6 (for now?) - I started this out because that's an output I needed and didn't have, but it wasn't in S6 at the time most likely. I can see how it's better suited for Stage 6 so I will remove the transmission spectrum code from functions. I'll post here when it's done.

Reply:
ok, it's done!